tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -825,7 +826,7 @@ def _post_fit(self, keep_only_best=False, refit_full=False, set_best_to_refit_fu
             self.delete_models(models_to_keep='best', dry_run=False)
 
         if calibrate:
```

Comment:
Update documentation for `calibrate` argument to describe what it does for problem_type='quantile' (also briefly describe what happens with bagging vs no bagging)

Reply:
Doesn't look like the documentation for `calibrate` argument has been updated?

The current `calibrate` documentation only mentions what happens for classification, not for what happens with `problem_type='quantile'`